CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+## v5.7.0 (2023-09-05)
+
+- Fix FedEx SmartPost carrier account creation not going to the correct endpoint
+- Add missing optional `ReturnAddress` and `BuyerAddress` parameters to Shipment creation parameter set
```

Comment:
nit for future: prefer listing breaking -> new -> fixes in that order of importance